src/components/map/Map.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import L from 'leaflet';
+import { Draw, Control } from 'leaflet-draw';
+
+const MAP_MIN_ZOOM = 2;
+const MAP_INITIAL_ZOOM = 3;
+const BASEMAP_TILE = 'http://server.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer/tile/{z}/{y}/{x}';
+const MAP_CENTER = [51.505, -0.09];
+const BASEMAP_ATTRIBUTION = 'Tiles &copy; Esri &mdash; Source: Esri, i-cubed, USDA, USGS, AEX, GeoEye, Getmapping, Aerogrid, IGN, IGP, UPR-EGP, and the GIS User Community';
+
+const activeLayers = new L.FeatureGroup();
```

Comment:
This should live inside Map class as a property. And I think the name is a little deceiving.